### DISCUSSION TITLE
My Account Positions

### DISCUSSION THREAD

**Discussion by nickjames640** [areed1192/td-ameritrade-python-api#185] [General]
Maybe its too early and my coffee hasn't worked yet, but do you have an API to get an account's positions? As in, the v1/accounts/accountNumber path?

Thanks

**Comment by eyesuk** [comment]
The code below will get your positions:

`accounts_data = TDSession.get_accounts(
    account='youraccountnumber',
    fields=['positions']
)`

**Reply by nickjames640** [reply]
thanks!
